core/src/torii/routing.rs

```diff
@@ -289,6 +289,50 @@ async fn update_metrics<W: WorldTrait>(
     Ok(())
 }
 
+/// Convert accumulated `Rejection` into appropriate `Reply`.
+#[allow(clippy::unused_async)]
+// TODO: -> Result<impl Reply, Infallible>
```

Comment:
I would add an issue for this.

Reply:
https://github.com/hyperledger/iroha/issues/1934